lib/message_generator.dart

```diff
@@ -257,6 +264,10 @@ class MessageGenerator extends ProtobufContainer {
             " checkItemFailed(v, '$classname');");
       });
       generateFieldsAccessorsMutators(out);
+      generateStaticUnpacker(out);
```

Comment:
I assume the Any class doesn't need an unpacker, this could be left as an else {} to generateAnyMethods